src/ConsoleLogger.cs

```diff
@@ -9,10 +9,12 @@ public enum LogLevel
         {
             None = 0,
             Debug = 1,
-            Warning = 2,
-            Error = 4,
-            Exception = 8,
-            All = Debug | Warning | Error | Exception
+            Trace = 2,
+            Info = 4,
+            Warning = 8,
+            Error = 16,
+            Exception = 32,
+            All = Debug | Trace | Info | Warning | Error | Exception

```

Comment:
I know it was a hardcoded list before, but at this point maybe just

```suggestion
            All = ~0
```

Reply:
Sure